sphinx/pycode/function.py

```diff
@@ -0,0 +1,124 @@
+"""
+    sphinx.pycode.function
+    ~~~~~~~~~~~~~~~~~~~~~~
+
+    Utilities parsing function definitions
+
+    :copyright: Copyright 2007-2020 by the Sphinx team, see AUTHORS.
+    :license: BSD, see LICENSE for details.
+"""
+
+import ast
+import sys
+from collections import OrderedDict
+from inspect import Parameter
+from typing import cast
+from typing import Mapping
+
+if sys.version_info > (3, 8):
+    from functools import cached_property
+else:
+    from cached_property import cached_property
+
+
+def stringify(node: ast.AST) -> str:
```

Comment:
`unparse` might be better - a future version of ast will have a builtin `unparse` member

Reply:
Good to hear. It's good naming!
https://mail.python.org/archives/list/python-dev@python.org/thread/JAQDBMC23HW2PQ27HQNJ7G244T423IDD/

Reply:
done.